executor/src/witgen/jit/processor.rs

```diff
@@ -267,20 +268,20 @@ impl<'a, T: FieldElement> Processor<'a, T> {
         Ok(result)
     }
 
-    fn process_until_no_progress<FixedEval: FixedEvaluator<T>>(
+    fn process_until_no_progress<'queue, FixedEval: FixedEvaluator<T>>(
         &self,
         can_process: impl CanProcessCall<T>,
         witgen: &mut WitgenInference<'a, T, FixedEval>,
-        mut identity_queue: IdentityQueue<'a, T>,
+        mut identity_queue: IdentityQueue<'a, 'queue, T>,
```

Comment:
```suggestion
        mut identity_queue: IdentityQueue<'a, '_, T>,
```
This seems to work, then you can remove the `'queue` parameter.